libs/sdk-ui-kit/src/Dialog/DialogBase.tsx

```diff
@@ -15,12 +15,13 @@ const checkKeyHandler = (event: React.KeyboardEvent, keyCode: number, handler: (
     }
 };
 
-const isTextInputComponent = ({ target }: React.KeyboardEvent): boolean => {
+const inputComponentHandler = ({ target }: React.KeyboardEvent): boolean => {
```

Comment:
The new name is not ideal, can we do something like `shouldSubmitOnEnterPress` or something like that? The key thing here is the name should suggest this returns a boolean.